### ISSUE TITLE
Erroneous error raised when referencing toml submodule

### ISSUE THREAD

**Issue by enpaul** [python/mypy#7686]
**Are you reporting a bug, or opening a feature request?**
* Reporting a bug

**Please insert below the code you are checking with mypy**

```python
import toml
from myapp import InvalidConfigError

try:
    with open("config.toml", "r") as infile:
        data = toml.load(infile, _dict=dict)
except toml.decoder.TomlDecodeError as err:
    raise InvalidConfigError(f"Invalid configuration file: {err}") from None
```

**What is the actual behavior/output?**

* Attribute error on the `toml` module:
```
myapp/myscript.py:7: error: Module has no attribute "decoder"
Found 1 error in 1 file (checked 10 source files)
```

**What is the behavior/output you expect?**

* No errors raised:
```
Success: no issues found in 10 source files
```

* The import is possible in interactive shell:
```
>>> import toml
>>> toml.decoder.TomlDecodeError
<class 'toml.decoder.TomlDecodeError'>
>>>
```

**What are the versions of mypy and Python you are using?**

```
> python --version
Python 3.7.4
> mypy --version
mypy 0.730
```

**Do you see the same issue after installing mypy from Git master?**

Yes, using `9614518856674c98e513828dab790d094a660587`

**What are the mypy flags you are using?**

```
--ignore-missing-imports --no-strict-optional
```

**Comment by JelleZijlstra** [comment]
This is an issue in typeshed: https://github.com/python/typeshed/blob/master/third_party/2and3/toml.pyi is missing the decoder file. You should file an issue in the typeshed repo to get this fixed. It looks like you could work around this by using `toml.TomlDecodeError` directly too.

**Comment by enpaul** [comment]
@JelleZijlstra Thank you very much for that clarification! I'll open an issue or pull request with typeshed.
